third_party/nvidia/backend/compiler.py

```diff
@@ -62,6 +62,7 @@ class CUDAOptions:
     ptx_version: int = None
     enable_fp_fusion: bool = True
     allow_fp8e4nv: bool = False
+    allow_tf32: bool = False
```

Comment:
I'm not sure about the name of this option.  Without the context in this patch, I'd think this means, do we allow implicitly upgrading fp32 dots to tf32, which is not what this controls.

Perhaps supports_tf32, with a comment saying that this indicates whether the hw supports tf32 but doesn't give us permission to *use* tf32 to silently, except where the result is "as if" we'd computed in fp32?

Reply:
Yeah, I wasn't sure about this name. I followed the convention of `allow_fp8e4nv`, but I agree that `supports_tf32` is a much better name. Let me change it.